packages/alfa-rules/src/common/predicate/is-clipped-by-size.ts

```diff
@@ -11,7 +11,7 @@ const { isText } = Text;
 
 const cache = Cache.empty<Device, Cache<Context, Cache<Node, boolean>>>();
 
-export function isClipped(
+export function isClippedBySize(
```

Comment:
Splitting these in two opens up the possibility to use one without the other, which really shouldn't ever happen. You can put them in the same module as two functions and then export an `or()` of the two.